toonz/sources/toonzqt/functionsheet.cpp

```diff
@@ -970,7 +970,7 @@ FunctionSheet::FunctionSheet(QWidget *parent, bool isFloating)
   setRowsPanel(m_rowViewer = new FunctionSheetRowViewer(this));
   setCellsPanel(m_cellViewer = new FunctionSheetCellViewer(this));
 
-  setWindowFlag(Qt::Window);
+  setWindowFlags(Qt::Window);
```

Comment:
ahh yes I see I was wondering whether or not it was intended or not looking at the two functions in the documentation. So I will try build without the line since debian stable only uses 5.7.1+dfsg-3+b1 at the moment.